### ISSUE TITLE
Error: unused argument (save_freq = "epoch")

### ISSUE THREAD

**Issue by asheetal** [rstudio/keras#899]
My pip3 package list is as follows

`
absl-py (0.8.1)
astor (0.8.0)
cffi (1.12.3)
enum34 (1.1.6)
gast (0.3.2)
google-pasta (0.1.7)
grpcio (1.24.1)
h5py (2.10.0)
Keras (2.2.5)
Keras-Applications (1.0.8)
Keras-Preprocessing (1.1.0)
Markdown (3.1.1)
numpy (1.17.2)
pip (9.0.3)
plaidml (0.6.4)
plaidml-keras (0.6.4)
protobuf (3.10.0)
pycparser (2.19)
PyYAML (5.1.2)
scipy (1.3.1)
setuptools (41.4.0)
six (1.12.0)
tensorboard (2.0.0)
tensorflow (1.14.0)
tensorflow-estimator (2.0.1)
termcolor (1.1.0)
Werkzeug (0.16.0)
wheel (0.33.6)
wrapt (1.11.2)
`
I am getting an error with this code below
`
callback_model_checkpoint(
       filepath = "file.hd5", 
       monitor='val_loss',
       mode='auto',
      save_freq = "epoch",
      save_best_only = FALSE, 
      verbose = 0  )`

> Error in (function (filepath, monitor = NULL, verbose = NULL, save_best_only = NULL,  : 
> unused argument (save_freq = "epoch")

Anyone seen this before? Maybe my packages are messed up. Hope someone can help

**Comment by dfalbel** [comment]
I can't reproduce locally, can you tell what's your `sessionInfo()` and `tf_version()`.

**Comment by asheetal** [comment]
```

> sessionInfo() R version 3.6.0 (2019-04-26) 
Platform: x86_64-redhat-linux-gnu (64-bit) 
Running under: CentOS Linux 7 (Core)  
Matrix products: default 
BLAS/LAPACK: /usr/lib64/R/lib/libRblas.so  
locale:  
[1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8      
[5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=en_US.UTF-8       LC_NAME=C                   
[9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C         
attached base packages: 
[1] parallel  stats     graphics  grDevices utils     datasets  methods   base       
other attached packages:  
[1] RCurl_1.95-4.12   bitops_1.0-6      httr_1.4.1        jsonlite_1.6      caret_6.0-84      ggplot2_3.2.1     lattice_0.20-38    
[8] stringi_1.4.3     readxl_1.3.1      filenamer_0.2.3   tibble_2.1.3      caTools_1.17.1.2  mltools_0.3.5     data.table_1.12.6 
[15] varhandle_2.0.4   dplyr_0.8.3       keras_2.2.5.0      
loaded via a namespace (and not attached):  
[1] Rcpp_1.0.2             lubridate_1.7.4        class_7.3-15           assertthat_0.2.1       zeallot_0.1.0           
[6] ipred_0.9-9            foreach_1.4.7          R6_2.4.0               cellranger_1.1.0       plyr_1.8.4             
[11] stats4_3.6.0           pillar_1.4.2           tfruns_1.4             rlang_0.4.0.9004       lazyeval_0.2.2         
[16] rstudioapi_0.10        whisker_0.4            rpart_4.1-15           Matrix_1.2-17          reticulate_1.13.0-9001 
[21] splines_3.6.0          gower_0.2.1            stringr_1.4.0          munsell_0.5.0          compiler_3.6.0         
[26] pkgconfig_2.0.3        base64enc_0.1-3        tensorflow_2.0.0       nnet_7.3-12            tidyselect_0.2.5       
[31] prodlim_2018.04.18     codetools_0.2-16       crayon_1.3.4           withr_2.1.2            MASS_7.3-51.4          
[36] recipes_0.1.7          rappdirs_0.3.1         ModelMetrics_1.2.2     grid_3.6.0             nlme_3.1-141           
[41] gtable_0.3.0           magrittr_1.5           scales_1.0.0           reshape2_1.4.3         timeDate_3043.102      
[46] generics_0.0.2         lava_1.6.6             iterators_1.0.12       tools_3.6.0            glue_1.3.1             
[51] purrr_0.3.3            survival_2.44-1.1      colorspace_1.4-1
--

```

I am using plaidml backend 0.6.4 . with following commands
`use_implementation("keras")
use_backend("plaidml")`

**Comment by asheetal** [comment]
Here is the code where I see the problem, just modified from the original MNIST cost

https://www.dropbox.com/s/n3rw1cya4nmu2ey/keras.R?dl=0

**Comment by dfalbel** [comment]
@asheetal I have pushed a fix in #903 . I'll merge when automated tests are OK. 
You can install it with `devtools::install_github("rstudio/keras#903")`.  However, please note that plaidml backend is very experimental in Keras, and support may be dropped in future versions.

**Comment by asheetal** [comment]
Wow that is amazing and so fast. Thanks a lot for your help. Will check out the new code. The issue is that I am doing university research on hairstring budget using cheap old AMD cards. And plaidml has been the only backend that I got to work. I hope there is a stronger backend alternative soon.
